src/main/java/nextstep/ladder/domain/LadderPlay.java

```diff
@@ -0,0 +1,40 @@
+package nextstep.ladder.domain;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class LadderPlay {
+    private static final String LADDER_GAME_ERROR = "결과 수와 참여자 수는 같아야 합니다.";
+    private final Players players;
```

Comment:
```suggestion
    private static final String LADDER_GAME_ERROR = "결과 수와 참여자 수는 같아야 합니다.";
    
    private final Players players;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다!